SimpleCalculator.Tests/EvaluateTests.cs

```diff
@@ -8,13 +8,20 @@ public class EvaluateTests
     {
         Evaluate TestEvaluate = new Evaluate();

```

Comment:
This line should be moved to the first test.  All tests need to start with a clean base.